android/app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,10 @@
             <category android:name="android.intent.category.BROWSABLE" />
             <data android:scheme="https" />
         </intent>
+        <intent>
+            <action android:name="android.intent.action.SEND" />
```

Comment:
wofür brauchen wir android.intent.action.SEND ?

Reply:
Um mailto: Links aus der App aufrufbar zu machen.
Wird in der "Über die App" Seite benötigt.